algorithms/scaling/observers.py

```diff
@@ -191,9 +197,37 @@ def print_scaling_summary(self, scaling_script):
             scaling_script.params.reflection_selection.min_partiality,
         )
         if MergingStatisticsObserver().data:
+            merging_stats = MergingStatisticsObserver().data["statistics"]
+            logger.info(make_merging_statistics_summary(merging_stats))
+            vals = cc_half_fit(merging_stats, limit=0.3)
+            r_cc = vals[0]
+            max_current_res = merging_stats.bins[-1].d_min
+            cut_merging_statistics_result = None
+            cut_anom_merging_statistics_result = None
+            if r_cc - max_current_res > 0.005:
```

Comment:
`0.005` because?

Reply:
Additional: I dislike magic numbers, and also the significance of these depends very much on the value under comparison

Reply:
We report resolution stats to 2dp, so this is meant to be "if the same to within two decimal places". xia2's magic number here is 0.004 🤷‍♂️ 🙃